### PR TITLE
Initial message functionality

### LINKED ISSUE #7: User can join chatroom and see a list of recent messages from that room

### PR DESCRIPTION
**Pull request by AllPurposeName**
closes #7 . Austin.

### CODE REVIEW

**Comment by kyrasteen** [comment]
I hate DJ.

File: app/models/room.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
 class Room < ActiveRecord::Base
+  after_create :slug_it_up
+  has_many :messages
+
+  def to_param
+    slug
+  end
+
+  def slug_it_up
```
Comment (indiesquidge) [inline app/models/room.rb:9]: This is such a stupid method name....😑

File: app/helpers/rooms_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module RoomsHelper
+end
```
Comment (indiesquidge) [inline app/helpers/rooms_helper.rb:2]: This file should have been deleted

File: app/models/room.rb (hunk 1 above)
Comment (AllPurposeName) [inline app/models/room.rb:9]: Go drink your H8rAde

**Pull request merged**